terraform/aws-constellation/variables.tf

```diff
@@ -27,7 +27,7 @@ variable "node_groups" {
     instance_type = string
     disk_size     = number
     disk_type     = string
-    zone          = string
+    zone          = optional(string, "")
```

Comment:
Is it correct that this field can be empty in the config? I see it makes sense to keep it optional and use var.zone otherwise though. Please add E2E runs for AWS and GCP also when ready